%YAML:1.0
---
num_cameras: 3
cameras:
   -
      focal: 1.0333413229124364e+04
      aspect: 1.
      ppx: 640.
      ppy: 360.
      R: !!opencv-matrix
         rows: 3
         cols: 3
         dt: f
         data: [ 9.97302175e-01, -3.07087158e-03, 7.33406618e-02,
             3.49279190e-03, 9.99978065e-01, -5.62470593e-03,
             -7.33217299e-02, 5.86572196e-03, 9.97291148e-01 ]
      t: !!opencv-matrix
         rows: 3
         cols: 1
         dt: d
         data: [ 0., 0., 0. ]
      K_matrix: !!opencv-matrix
         rows: 3
         cols: 3
         dt: d
         data: [ 1.0333413229124364e+04, 0., 640., 0.,
             1.0333413229124364e+04, 360., 0., 0., 1. ]
   -
      focal: 1.0279708009849917e+04
      aspect: 1.
      ppx: 640.
      ppy: 360.
      R: !!opencv-matrix
         rows: 3
         cols: 3
         dt: f
         data: [ 1., 1.03168896e-08, 9.12656972e-09, -8.27577296e-09, 1.,
             -5.46630119e-09, 9.12656972e-09, 3.79687481e-09, 1. ]
      t: !!opencv-matrix
         rows: 3
         cols: 1
         dt: d
         data: [ 0., 0., 0. ]
      K_matrix: !!opencv-matrix
         rows: 3
         cols: 3
         dt: d
         data: [ 1.0279708009849917e+04, 0., 640., 0.,
             1.0279708009849917e+04, 360., 0., 0., 1. ]
   -
      focal: 9.9717829180837743e+03
      aspect: 1.
      ppx: 640.
      ppy: 360.
      R: !!opencv-matrix
         rows: 3
         cols: 3
         dt: f
         data: [ 9.96445596e-01, 1.00272382e-02, -8.36398229e-02,
             -1.03565129e-02, 9.99940217e-01, -3.50443996e-03,
             8.35997090e-02, 4.35821759e-03, 9.96489823e-01 ]
      t: !!opencv-matrix
         rows: 3
         cols: 1
         dt: d
         data: [ 0., 0., 0. ]
      K_matrix: !!opencv-matrix
         rows: 3
         cols: 3
         dt: d
         data: [ 9.9717829180837743e+03, 0., 640., 0.,
             9.9717829180837743e+03, 360., 0., 0., 1. ]
